.github/workflows/build-test-package.yml

```diff
@@ -11,10 +11,10 @@ on:
       - main
 
 jobs:
-  cxx-build-workflow:
+  cxx:
     uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-cxx.yml@v5.4.2
 
-  python-build-workflow-dev:
+  python-dev:
```

Comment:
Much better already, but still too long to distinguish all checks:

    Build / python-dev / build-windows-python-package...
    Build / python-dev / build-windows-python-package...

See:
![image](https://github.com/user-attachments/assets/1f202d83-e662-4d59-b194-a6661fe31b3b)